nais/naiserator.yml

```diff
@@ -36,15 +36,16 @@ spec:
   prometheus:
     enabled: true
     path: /metrics
-  vault:
-    enabled: true
-    paths:
-      - mountPath: /var/run/secrets/nais.io/vault
-        kvPath: {{vaultKvPath}}
-      - mountPath: /var/run/secrets/nais.io/azuread
-        kvPath: {{azureadKvPath}}
-      - mountPath: /var/run/secrets/nais.io/serviceuser
-        kvPath: {{serviceuserKvPath}}
+  azure:
+    application:
+      enabled: true
+      tenant: {{tenant}}
+    accessPolicy:
```

Comment:
Ligger denne kanskje en indent for mye inn?

Reply:
Nice catch. Ja, accessPolicy burde ligge på lik linje med azure.